gsl_notification/templates/gsl_notification/tab_simulation_projet/tab_notifications.html

```diff
@@ -70,9 +70,15 @@ <h2>
                                                 </a>
                                             </li>
                                             <li>
-                                                <button class="fr-btn fr-btn--secondary arrete-card__action" disabled>
-                                                    Supprimer
-                                                </button>
+                                                <form method="post"
+                                                      action="{% url 'gsl_notification:delete-arrete' document.id %}">
+                                                    {% csrf_token %}
+                                                    <button type="submit"
+                                                            class="fr-btn fr-btn--secondary arrete-card__action arrete-delete-button"
+                                                            onclick="return confirm('Êtes-vous sûr de vouloir supprimer cet arrêté ?');">
```

Comment:
Le onclick est bloqué par la CSP : 

```suggestion
                                                            class="fr-btn fr-btn--secondary arrete-card__action arrete-delete-button">
```